tools/requirements.txt

```diff
@@ -1,6 +1,6 @@
 pygit2==1.0.1
 bidict==0.13.1
-Pillow==9.0.0
+Pillow==8.4.0
```

Comment:
atomization, this is already in a different PR

Reply:
I literally cannot make commits without this change.